source/index.txt

```diff
@@ -6,15 +6,15 @@ Cluster-to-Cluster Sync
 
 .. default-domain:: mongodb
 
-{+c2c-product-name+} provides you with continuous unidirectional
-data synchronization between two MongoDB clusters in the same or
-hybrid environments. Customers can enable {+c2c-product-name+} via
-the :ref:`c2c-mongosync` utility tool.
+{+c2c-product-name+} provides continuous data synchronization or a 
```

Comment:
This is an unrelated fix  -- the text at left got messed up merging the docs in this PR